src/main/java/dumshenko/daniil/Shape.java

```diff
@@ -0,0 +1,26 @@
+package dumshenko.daniil;
+
+enum FigureType {
```

Comment:
nit: Probably a bit better to name this enum `ShapeType`. Also let's extract this class into separate file.
```suggestion
enum ShapeType {
```